core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/event/ClusterCertificatesRotationTriggerEvent.java

```diff
@@ -23,4 +25,13 @@ public ClusterCertificatesRotationTriggerEvent(String event, Long resourceId, Pr
     public CertificateRotationType getCertificateRotationType() {
         return certificateRotationType;
     }
+
+    @Override
+    public boolean equalsEvent(StackEvent other) {
+        if (!isClassAndEqualsEvent(ClusterCertificatesRotationTriggerEvent.class, other)) {
+            return false;
+        }
```

Comment:
this part is very similar in every `equalsEvent` method, maybe it is worth to extract it in `StackEvent`

Reply:
or event specific comparisons should be extracted into different overridden method